### ISSUE TITLE
Applied Energistics 2 Client Java Error

### ISSUE THREAD

**Issue by Enkelschrecken** [FTBTeam/FTB-Continuum#383]
* **Modpack Version**: FTB Continuum Version: 1.4.1

* **Issue**: Java.internal.io.exception when accessing the Me System. I takes a while until I get kicked out of the server. My friends do not have that issue and reinstalling does not help. The Issue is on the client side because when I get this massive lag the others can play normal. It is all fine, every mod works I can open other inventories but once I open the Me System the lag or kick out is there.

* **Is it Repeatable?**: Yes every time I open the Me System.

* **Mod/s Affected**: Applied Energistics 2

**Comment by almightyfoon** [comment]
Have you tried updating java or increasing the ammount of memory you have for your client?

**Comment by Enkelschrecken** [comment]
Yes I installed everything new but no change. We found out that the storage busses from AE2 were the problem. Our main storage is barrels with a barrel connector and storage bus. We tought that would cause the lag because it trys to send me a lot of item data when I open the terminal. But the problem was the fluid storage bus in combination with Demonic Drums, the largest. After opening the terminal fluid storage bus sent me so much data from those drums that it was like a DDOS for my Minecraft/PC.
